src/main/kotlin/app/ui/AddRepoState.kt

```diff
@@ -4,30 +4,31 @@
 package app.ui
 
 import app.Configurator
-import app.model.Repo
+import app.model.LocalRepo
 import app.utils.RepoHelper
 
 /**
  * Add repository dialog console UI state.
  */
 class AddRepoState constructor(val context: Context) : ConsoleState {
     override fun doAction() {
-        if (Configurator.getRepos().isEmpty()) {
+        if (Configurator.getLocalRepos().isEmpty()) {
```

Comment:
15/10
it seems you want to beat a record of number of indentation. Why not `if (...) return`